src/tvmlkit.cs

```diff
@@ -120,6 +120,7 @@ public enum TVImageType : long {
 	[ErrorDomain ("TVMLKitErrorDomain")]
 	public enum TVMLKitError : long {
 		Unknown = 1,
+		InvalidArguments,
```

Comment:
The order of the enums matter. This changes makes InternetUnavailable to be the old one +1. Which is bad it the users do depend on the casting of enums to long/

Reply:
It's basically an ABI break.

Reply:
Not just enum to long. It breaks interop too.